gdal/frmts/pdf/pdfcreatecopy.cpp

```diff
@@ -2370,29 +2485,41 @@ int GDALPDFWriter::WriteOGRFeature(GDALPDFLayerDesc& osVectorDesc,
                     }
                 }
 
+                pszStr = OGR_ST_GetParamStr(hTool, OGRSTLabelFontName, &bIsNull);
+                if (pszStr && !bIsNull)
+                    osTextFont = pszStr;
+
                 double dfVal = OGR_ST_GetParamDbl(hTool, OGRSTLabelSize, &bIsNull);
                 if (!bIsNull)
-                {
                     dfTextSize = dfVal;
-                }
 
                 dfVal = OGR_ST_GetParamDbl(hTool, OGRSTLabelAngle, &bIsNull);
                 if (!bIsNull)
-                {
-                    dfTextAngle = dfVal;
-                }
+                    dfTextAngle = dfVal * M_PI / 180.0;
+
+                dfVal = OGR_ST_GetParamDbl(hTool, OGRSTLabelStretch, &bIsNull);
+                if (!bIsNull)
+                    dfTextStretch = dfVal / 100.0;
 
                 dfVal = OGR_ST_GetParamDbl(hTool, OGRSTLabelDx, &bIsNull);
                 if (!bIsNull)
-                {
                     dfTextDx = dfVal;
-                }
 
                 dfVal = OGR_ST_GetParamDbl(hTool, OGRSTLabelDy, &bIsNull);
                 if (!bIsNull)
-                {
                     dfTextDy = dfVal;
-                }
+
+                int nVal = OGR_ST_GetParamNum(hTool, OGRSTLabelAnchor, &bIsNull);
+                if (!bIsNull)
+                    nTextAnchor = nVal;
+
+                nVal = OGR_ST_GetParamNum(hTool, OGRSTLabelBold, &bIsNull);
+                if (!bIsNull)
+                    bTextBold = (nVal != 0); // TODO clarify in feature style spec that bo:0 = not bold, any other integer = bold
```

Comment:
From what I understand from the code, the serialization should be like LABEL(....,bo,....) and in OGRStylePool::Parse(), the value should be set to 1.

Reply:
> the colon should always be included, and any value other than numeric 0 should be considered "true" (or perhaps specify that the only valid values are numeric 0 and numeric 1).

Looks reasonable to me. Feel free to edit ogr_feature_style.html in that direction